_infra/helm/locust/locustfiles/requests.json

```diff
@@ -0,0 +1,19 @@
+{
+    "requests": [
+        {
+            "method": "GET",
+            "url": "/surveys/todo",
+            "expected_response_text": "Click on the survey name to complete your questionnaire"
+        },
+        {
+            "method": "GET",
+            "url": "/surveys/history",
+            "expected_response_text": "aria-selected=\"true\" id=\"SURVEY_HISTORY_TAB\" role=\"tab\">History</a>"
+        },
+        {
+            "method": "GET",
+            "url": "/secure-message/threads",
+            "expected_response_text": "No new conversations"
```

Comment:
The rendered page contains
```
<title>Messages | ONS Business Surveys</title>
```
which we should probably assert on. "No new conversations" is only true due to us knowing where this request is in the journey we're introducing. As we introduce other journeys we may need to assert on specific text like that, but for these initial clicks we probably just want something fairly static on the page.

Reply:
Done.